src/clang-cpp-frontend/clang_cpp_adjust.h

```diff
@@ -32,6 +41,37 @@ class clang_cpp_adjust : public clang_c_adjust
   void adjust_side_effect(side_effect_exprt &expr) override;
   void adjust_new(exprt &expr);
   void adjust_struct_method_call(member_exprt &expr);
+
+  /**
+   * methods for implicit GOTO code generation
+   */
+  void gen_vptr_initializations(symbolt &symbol);
```

Comment:
How is "implicit GOTO code generation" related to "vptr initializations"? Could you be more concrete about the functionality here? Again, it's commenting just on a single method.

Reply:
> How is "implicit GOTO code generation" related to "vptr initializations"? Could you be more concrete about the functionality here?

Done, improved the comment.